design/collaborators.go

```diff
@@ -0,0 +1,52 @@
+package design
+
+import (
+	d "github.com/goadesign/goa/design"
+	a "github.com/goadesign/goa/design/apidsl"
+)
+
+var _ = a.Resource("collaborators", func() {
```

Comment:
I guess space-areas and other space-* resources were introduced because there were old "areas", "iterations" etc. So, space-* are new versions for space based resources. There is only one (already space based) collaborators resource.